packages/js/client/src/PolywrapClient.ts

```diff
@@ -171,6 +172,16 @@ export class PolywrapClient implements Client {
     );
   }
 
+  @Tracer.traceMethod("PolywrapClient: getPluginConfig")
+  public async getPluginConfig<TUri extends Uri | string>(
+    uri: TUri,
+    options: GetPluginConfigOptions = {}
```

Comment:
Since we are using options to interact with the `getPlugin` function, I think it would make sense to re-use the already existing type `GetPluginOptions`